other-docs/guides/getting-help-with-altis.md

```diff
@@ -0,0 +1,47 @@
+# Getting help with Altis.
+
+If you need help, this guide will show you how to most effectively get you the help you need.
+
+Well we support a few methods, the primary method and the route you should choose, is the Altis Dashboard Support button.
+
+<img width="1398" alt="Screenshot 2021-03-30 at 14 44 16" src="https://user-images.githubusercontent.com/1263370/113014595-a119f980-9174-11eb-95d6-0e2a9718c5e5.png">
+
+We break down support into different types. Each have their own language and proccesses. But they're somewhat intuitive once you internalise the language a little. Let's go over the types.
+
+<img width="1282" alt="Screenshot-2021-01-26-at-10 14 18" src="https://user-images.githubusercontent.com/1263370/113014673-b2630600-9174-11eb-88ae-ed3cc6b98078.png">
+
+In the above you can see we currently have four types of support. I’ll break down how they differ from eachother.
+
+# Incident
+
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an incident, and requires investigation.
+
+From the support side, Incidents are the things that make us scamble to respond and jump on the problem. If the site is down, this is an Incident, with a prioriy of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means neccessary.
+
+Incidents focus on immediate attention, with an emphisis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
```

Comment:
typo

I might suggest emphasizing _immediate attention_, i.e. this is an urgent, drop-everything type problem, but it's up to you.

```suggestion
Incidents focus on immediate attention, with an emphasis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
```